src/FlowMeter.h

```diff
@@ -81,6 +83,8 @@ class FlowMeter {
     void tick(unsigned long duration = 1000);
     void count();                                //!< Increments the internal pulse counter. Serves as an interrupt callback routine.
     void reset();                                //!< Prepares the flow meter for a fresh measurement. Resets all current values, but not the totals.
+    void pause();
```

Comment:
I'm struggling to understand why someone would want to pause and resume *internal* counting during normal operation, except in order to 'cook the books'. ;)

Again, this might be a special needs change, tailored to your particular application. 

Could the feature you're implementing with these methods probably also be implemented using the original `reset()` method, leveraging the difference between current and total values? 
That may even have been my original intention of the `reset()` method: discard a counted amount of flow/volume (while still preserving internal total count).

Reply:
Yeah, another application specific change. I think I was having trouble with the original API so my assumption was that I had misunderstood the use of the original API. Lemme review that firmware and get back to you. Again, it was over a year ago 😅

Reply:
Okay, thanks. :)